gap/FunctorCategories.gi

```diff
@@ -821,8 +848,8 @@ InstallMethodWithCache( FunctorCategory,
                     fi;
                     
                     if not IsBound( images_of_objects[pos] ) then
-                        L := List( eval_arg, F -> ApplyCell( F, objB ) );
-                        images_of_objects[pos] := CallFuncList( oper, L );
+                        L := List( [ underlying_arguments ], cat_or_F_or_eta -> ApplyCell( cat_or_F_or_eta, objB ) );
```

Comment:
Locally deactivating caching by switching this line with the above `if`-line introduces a huge regression in `CatReps/examples/CategoryOfRepresentations.g`.

Reply:
Would it make sense to add this finding as a comment in the code?

Reply:
Done with 1503b10e.